libs/types.ts

```diff
@@ -17,4 +19,25 @@ interface Commitment {
   blockNumber: number;
 }
 
-export default Commitment;
+interface Transaction {
+  fee: string;
+  historicRootBlockNumberL2: string[];
+  transactionType: string;
+  tokenType: string;
+  tokenId: string;
+  value: string;
+  ercAddress: string;
+  recipientAddress: string;
+  commitments: string[];
+  nullifiers: string[];
+  compressedSecrets: string[];
+  proof: string[]; //string of hashes
+  transactionHash: string;
+}
```

Comment:
Maybe this is NightfallTransaction? Im debating between `Nightfall | Eth` vs `L1 | L2`. Let's discuss later.

Reply:
Is the nightfall transaction.